### ISSUE TITLE
Publish snap to store

### ISSUE THREAD

**Issue by superm1** [Eugeny/terminus#751]
Have you considered adding something to your release process to release the snap to https://snapcraft.io/?

It would be nice to have it automatically upgrading :)

**Comment by Xstoudi** [comment]
Yeah, especially considering that we now have to use the --dangerous flag ton install the snap file.

**Comment by superm1** [comment]
It's worth mentioning, this can be accomplished _directly_ with Circle CI.  Here's a sample of how it's done for the fwupd project: https://github.com/hughsie/fwupd/blob/master/.circleci/config.yml#L31

**Comment by superm1** [comment]
This is still something that would be quite useful.

**Comment by superm1** [comment]
still relevant
